Baby Monitor/Source Files/Custom Views/BaseViewController.swift

```diff
@@ -10,15 +10,34 @@ import UIKit
 /// In the future it will probably have more features.
 class BaseViewController: UIViewController {
 
-    init() {
+    let analytics: AnalyticsManager?
+
+    let analyticsScreenType: AnalyticsScreenType?
+
+    var className: String {
+        return String(describing: type(of: self))
+    }
+
+    init(analytics: AnalyticsManager?, analyticsScreenType: AnalyticsScreenType?) {
```

Comment:
"Write documentation for every method and property accessible outside the class."
Missing documentation in BaseViewController.